src/main/java/org/folio/inventory/validation/InstancesValidators.java

```diff
@@ -0,0 +1,71 @@
+package org.folio.inventory.validation;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.domain.instances.titles.PrecedingSucceedingTitle;
+import org.folio.inventory.support.http.server.ValidationError;
+import org.folio.inventory.exceptions.NotFoundException;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+
+public final class InstancesValidators {
```

Comment:
Could this class be specific to (preceding or succeeding) unconnected titles, rather than instances in general?

That might mean some of these method names could be significantly simplified.

The methods could still take an instance as a parameter

Reply:
This class contains some method that are not specific for preceding/succeeding titles: `refuseWhenInstanceNotFound`, `refuseWhenHridChanged`.

I can extract methods that are used for titles validation into a separate class, does it make sence?

Reply:
I think it makes sense for those to be in separate classes

Reply:
Extracted to `InstancePrecedingSucceedingTitleValidators` class.